test/src/xquery/maps/mapsLookup.xql

```diff
@@ -0,0 +1,23 @@
+xquery version "1.0";
```

Comment:
Just confirmed that with your now-merged PR, my test query produces the proper warning. Nice job @shabanovd!

```xquery
xquery version "1.0";

map { "a": "b" }
```

> Cannot compile xquery: exerr:EXXQDY0003 Map is not available before XQuery 3.0 [source: xquery version "1.0"; map { "a": "b" }]